zilliqa/src/consensus.rs

```diff
@@ -414,6 +414,19 @@ impl Consensus {
         let next_exponential_backoff_timeout =
             consensus_timeout_ms * 2u64.pow((view_difference + 1) as u32);
```

Comment:
I forgot to fix this when I changed the formula in `fn get_consensus_timeout_params(`). The correst formula to be used here is: `consensus_timeout_ms * 2u64.pow((view_difference + 1 as u32).saturating_sub(2));`

Reply:
Maybe move the 3 lines from above, setting `view_difference`, `consensus_timeout_ms` and `next_exponential_backoff_timeout` right above the `!info()` statement below, since the 3 variables are only used there

Reply:
Good points, thank you.